build.gradle

```diff
@@ -58,12 +58,17 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 
+	implementation 'io.micrometer:micrometer-tracing'
+	implementation 'io.micrometer:micrometer-tracing-bridge-otel'
+	implementation 'io.micrometer:micrometer-registry-otlp'
+
+	implementation platform('io.opentelemetry:opentelemetry-bom:1.38.0')
```

Comment:
If there is only a single otel dependency to add, is it necessary to add a BOM?

Reply:
the bom is not necessary. the version is dictated by another bom